tests/e2e/v3_curl_maxstream_test.go

```diff
@@ -162,22 +150,49 @@ func submitConcurrentWatch(cx ctlCtx, number int, wgDone *sync.WaitGroup, errCh
 	var wgSchedule sync.WaitGroup
 	wgSchedule.Add(number)
```

Comment:
It seems that we can remove `wgSchedule`, there is no reason to keep it because once it gets the `"created":true` response, then the goroutine must have been scheduled.

Reply:
Thanks for the review! 

The `wgSchedule` is used to make sure that all the goroutines are ready. If we remove this, we need to setup curl command one by one. It might take a long time to setup the ton of connections. I think we can keep the `wgSchedule` and run case in the efficient way. WDYT?

Reply:
Please consider to add a timeout, because `submitConcurrentWatch` may be blocked for a long time. Probably we can refactor the `submitConcurrentWatch` using `ExecuteUntil`. FYI. https://github.com/etcd-io/etcd/blob/5344085338a8dc57eb87f8676dea66401185b362/tests/framework/testutils/execute.go#L32

Reply:
Thanks for the hint. Let me try it.

Reply:
Updated with testutilslExecuteWithTimeout and the timeout is from ctlCtx.timeout